docs/examples/python-requests-jwt.md

```diff
@@ -0,0 +1,49 @@
+## Accessing PostgREST API using Python Requests and Requests-JWT
+
+This code relies on setting up the PostgreSQL auth functions and grants correctly first. Follow these instructions:
+
+http://postgrest.com/examples/users/
+
+After completing the configuration, be sure to create a user with email, password, role, and verified flag. We'll use that user to login in the code below.
+
+Install the required libraries
+
+    pip install requests
+    pip install requests-jwt
+
+Then, from a python interpreter or script:
+
+    import requests
+    import requests_jwt
+    import json
+
+    # email and password auth through postgrest
+    resp = requests.post('http://localhost:3000/rpc/login', json={"email": "you@yours.com", "pass": "dog"})
+    if resp.status_code != 200:
+        raise Exception()
+
+    # JWT auth using the token above
+    token = json.loads(resp.text)['token']
+    auth = requests_jwt.JWTAuth(token)
+    if r.status_code != 200:
+        raise Exception()
+
+    # Make a request to get the range header
+    r = requests.get('http://localhost:3000/posts', auth=auth)
```

Comment:
I was thinking the same thing, except that my initial thought, intuitively, was: why don't I just make a HEAD request? Then I made the HEAD request, saw that it was neutered, read the HTTP docs at w3, and saw that the behavior with regard to Content-Range and payload was according to the docs and sat there puzzled, since it seemed like the perfect use of a HEAD request, since the count is a lightweight thing and I don't actually need the payload there, or further, may want to do something else if the count is a billion. I didn't actually understand why Fielding et al made that design choice? And why using HEAD this way is not an appropriate choice here?

Could I just make the get request with 0 range?